### PR TITLE
PloneFormGen improvements

### PR DESCRIPTION
**Pull request by maethu**
Several display issues has been resolved...

### CODE REVIEW

File: plonetheme/blueberry/scss/ploneformgen.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    .required {
+      margin-left: $margin-horizontal / 2;
+    }
+
+    > .fieldErrorBox {
+      display: block;
+      visibility: hidden;
+
+      @extend .visualClear;
```
Comment (bierik) [inline plonetheme/blueberry/scss/ploneformgen.scss]: Pls use `@include clearfix()`

**Comment by bierik** [comment]
Could you place some screenies?

**Comment by maethu** [comment]
Fixed several issues on the add widget form.
screen shot 2016-05-12 at 09 59 04

Fixed several display issues on the rendered/quickedit form like:
- Show label + required sign on one line
- Show input element below the label and not on the same line
- style fieldset differently for formgen, since there was no visual sign for the fieldset.
- Fix buttons margin. 

screen shot 2016-05-12 at 10 31 04

File: plonetheme/blueberry/scss/ploneformgen.scss (hunk 1 above)
Comment (maethu) [inline plonetheme/blueberry/scss/ploneformgen.scss]: @bierik don

**Comment by bierik** [comment]
👍  thank you

**Pull request merged**